### ISSUE TITLE
[5.0] Schemaorg links broken accessibility

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#41229]
### Steps to reproduce the issue
setup and configure the new schemaorg plugins. Open an article for editing and check the info box for any of the schema

### Expected result
Web links that are accessible and do not break WCAG standards

### Actual result

![image](https://github.com/joomla/joomla-cms/assets/1296369/cb8f6849-49c8-448f-9caf-5275335143ce)

As you can see the link fails
- use clear and concise **link** text that describe the destination
- make sure links take you to where you’d expect
Do not:
have links that are not clear on destination such as ‘click here’ or ‘read more’

**Comment by brianteeman** [comment]
closed see #41441
